script/client/spring/application.yml

```diff
@@ -59,18 +59,20 @@ seata:
     enable-client-batch-send-request: true
   config:
     type: file
+    file:
+      name: file.conf
```

Comment:
I think it should be removed, when using yml file is file.conf is not needed, it is ambiguous to put it here.

Reply:
OK, I removed the `config.file.name`, `registry.file.name`, `ConfigFileProperties.java`, `RegistryFileProperties.java`